Play 2.5.x support

Running example app _play-angular-standalone_ with `play-2.5.0` throws the Exception

```scala
play.api.http.HttpErrorHandlerExceptions$$anon$1: Execution exception[[RuntimeException: java.lang.NoSuchMethodError: play.api.libs.json.JsLookup$.$bslash$extension(Lplay/api/libs/json/JsLookupResult;Ljava/lang/String;)Lplay/api/libs/json/JsLookupResult;]]
	at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:269)
	at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:195)
	at play.api.GlobalSettings$class.onError(GlobalSettings.scala:160)
	at play.api.DefaultGlobal$.onError(GlobalSettings.scala:188)
	at play.api.http.GlobalSettingsHttpErrorHandler.onServerError(HttpErrorHandler.scala:98)
	at play.core.server.netty.PlayRequestHandler$$anonfun$2$$anonfun$apply$1.applyOrElse(PlayRequestHandler.scala:99)
	at play.core.server.netty.PlayRequestHandler$$anonfun$2$$anonfun$apply$1.applyOrElse(PlayRequestHandler.scala:98)
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:344)
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:343)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
Caused by: java.lang.RuntimeException: java.lang.NoSuchMethodError: play.api.libs.json.JsLookup$.$bslash$extension(Lplay/api/libs/json/JsLookupResult;Ljava/lang/String;)Lplay/api/libs/json/JsLookupResult;
	at play.api.mvc.ActionBuilder$$anon$2.apply(Action.scala:463)
	at play.api.mvc.Action$$anonfun$apply$2$$anonfun$apply$5$$anonfun$apply$6.apply(Action.scala:112)
	at play.api.mvc.Action$$anonfun$apply$2$$anonfun$apply$5$$anonfun$apply$6.apply(Action.scala:112)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21)
	at play.api.mvc.Action$$anonfun$apply$2$$anonfun$apply$5.apply(Action.scala:111)
	at play.api.mvc.Action$$anonfun$apply$2$$anonfun$apply$5.apply(Action.scala:110)
	at scala.Option.map(Option.scala:146)
	at play.api.mvc.Action$$anonfun$apply$2.apply(Action.scala:110)
	at play.api.mvc.Action$$anonfun$apply$2.apply(Action.scala:103)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
Caused by: java.lang.NoSuchMethodError: play.api.libs.json.JsLookup$.$bslash$extension(Lplay/api/libs/json/JsLookupResult;Ljava/lang/String;)Lplay/api/libs/json/JsLookupResult;
	at pdi.jwt.JwtJson$.getAlgorithm(JwtJson.scala:19)
	at pdi.jwt.JwtJson$.getAlgorithm(JwtJson.scala:14)
	at pdi.jwt.JwtJsonCommon$class.encode(JwtJsonCommon.scala:24)
	at pdi.jwt.JwtJson$.encode(JwtJson.scala:14)
	at pdi.jwt.JwtSession.serialize(JwtSession.scala:61)
	at pdi.jwt.JwtPlayImplicits$RichResult.withJwtSession(JwtPlayImplicits.scala:61)
	at pdi.jwt.JwtPlayImplicits$RichResult.addingToJwtSession(JwtPlayImplicits.scala:82)
	at controllers.Application$$anonfun$login$1$$anonfun$apply$2.apply(Application.scala:30)
	at controllers.Application$$anonfun$login$1$$anonfun$apply$2.apply(Application.scala:28)
	at play.api.libs.json.JsResult$class.fold(JsResult.scala:72)
```

--- a/build.sbt
+++ b/build.sbt
@@ -1,7 +1,7 @@
 import sbt._
 import Keys._
 import Tests._
-import play.Play.autoImport._
+import play.sbt.Play.autoImport._
 import PlayKeys._
 import Dependencies._
 import com.typesafe.sbt.SbtSite.SiteKeys._
@@ -34,7 +34,7 @@ pushScript := {
 val commonSettings = Seq(
   organization := "com.pauldijou",
   version := buildVersion,
-  scalaVersion := "2.11.6",
+  scalaVersion := "2.11.7",
   autoAPIMappings := true,
   crossScalaVersions := Seq("2.10.5", "2.11.6"),
   crossVersion := CrossVersion.binary,
@@ -298,6 +298,6 @@ lazy val examplePlayAngularProject = project.in(file("examples/play-angular"))
     name := "playAngular",
     routesGenerator := play.sbt.routes.RoutesKeys.InjectedRoutesGenerator
   )
-  .enablePlugins(play.PlayScala)
+  .enablePlugins(PlayScala)
   .aggregate(playEdge)
   .dependsOn(playEdge)

--- a/project/Dependencies.scala
+++ b/project/Dependencies.scala
@@ -2,7 +2,7 @@ import sbt._
 
 object Dependencies {
   object V {
-    val play = "2.4.6"
+    val play = "2.5.0"
     val json4s = "3.3.0"
     val circe = "0.2.1"
     val scalatest = "2.2.4"

--- a/project/build.properties
+++ b/project/build.properties
@@ -1,1 +1,1 @@
-sbt.version=0.13.8
+sbt.version=0.13.11

--- a/project/plugins.sbt
+++ b/project/plugins.sbt
@@ -9,7 +9,7 @@ resolvers ++= Seq(
   "Typesafe repository releases" at "http://repo.typesafe.com/typesafe/releases/"
 )
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.4.0")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.5.0")
 
 addSbtPlugin("com.github.gseitz" % "sbt-release"            % "0.7.1")
 addSbtPlugin("com.typesafe.sbt"  % "sbt-site"               % "0.8.1")

